deepmd/main.py

```diff
@@ -424,29 +424,30 @@ def main_parser() -> argparse.ArgumentParser:
     parser_compress = subparsers.add_parser(
         "compress",
         parents=[parser_log, parser_mpi_log],
-        help="(Supported backend: TensorFlow) compress a model",
+        help="Compress a model",
         formatter_class=RawTextArgumentDefaultsHelpFormatter,
         epilog=textwrap.dedent(
             """\
         examples:
             dp compress
-            dp compress -i graph.pb -o compressed.pb
+            dp compress -i graph -o compressed
+            dp --pt compress -i frozen_model -o compressed_model
         """
         ),
     )
     parser_compress.add_argument(
         "-i",
         "--input",
-        default="frozen_model.pb",
+        default="frozen_model",
         type=str,
-        help="The original frozen model, which will be compressed by the code",
+        help="The original frozen model, which will be compressed by the code. Filename (prefix) of the input model file. TensorFlow backend: suffix is .pb; PyTorch backend: suffix is .pth",
     )
     parser_compress.add_argument(
         "-o",
         "--output",
-        default="frozen_model_compressed.pb",
+        default="compressed_model",
```

Comment:
I am going to revert it to `frozen_model_compressed`. The main reason is that DP-GEN has relied on the default value of `--output`...

https://github.com/deepmodeling/dpgen/blob/0f7e15c99d05c4bb83db7797956d611cbc1dc689/dpgen/generator/run.py#L853

```suggestion
        default="frozen_model_compressed",
```